_test/test_transformation/test_ortho_proj_methods.m

```diff
@@ -355,15 +355,25 @@ function test_binning_range_the_same_4D(~)
         %------------------------------------------------------------------
         %
         function test_cut_dnd(this)
-            skipTest('waits for cut_dnd being refactored, ticket #796')
+            ws = warning('off','HORACE:realign_bin_edges:invalid_argument');
```

Comment:
because it fixed (changed) by lattice realignment procedure and when changes are big, it warns you. May be worth just printing the red message -- was not sure about this. Warning is easy to debug anyway